<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Login</class>
 <widget class="QWidget" name="Form_Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>欢迎登录学生通讯录系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Load">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Register">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>131</y>
     <width>82</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="Edit_UserName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_Password">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
